server/liveblog/themes/themes_assets/classic/main.js

```diff
@@ -32,6 +33,30 @@
             loading: true,
             finished: false,
             showAuthor: SHOW_AUTHOR,
+            comment: {
```

Comment:
This timeline controller is already quite big, I would have created a special controller for Comments, with also a new directive to combine controller and view. What do you think?